### DISCUSSION TITLE
198

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#177] [General]
# 潮流周刊第198期 - 千层金树

封面图拍摄于朋友送的千层金，插在瓶子里面宛如一颗树的感觉，非常茂盛，给家里添加了不少生机，挺好的。

https://weekly.tw93.fun/posts/198-%E5%8D%83%E5%B1%82%E9%87%91%E6%A0%91

**Comment by siyi98** [comment]
我是学生，搓澡泥可以送我一个吗

**Reply by tw93** [reply]
哈哈哈哈

**Comment by SimFG** [comment]
博主买了国内服务器部署这个吗，还是服务器在海外然后绑定国内的域名

**Reply by tw93** [reply]
vercel就好，绑定域名

**Reply by SimFG** [reply]
这样的话，网站是不是就没有备案，哈哈哈

**Reply by tw93** [reply]
哈哈，毕竟不是局域网..
